pfunk/_resultsdb.py

```diff
@@ -0,0 +1,162 @@
+#
+# Results DB module.
+#
+# This file is part of Pints Functional Testing.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the Pints
+#  functional testing software package.
+#
+
+from __future__ import print_function
```

Comment:
Not required, but we usually add

```
from __future__ import absolute_import, division
from __future__ import print_function, unicode_literals
```

so that we don't need to think about it any further